docs/rst/use_cases/large_data/large_data.rst

```diff
@@ -0,0 +1,266 @@
+.. _uses_cases_rosbot_xl_autonomy:
+
+Large data communication with ROS 2 (a practical example)
+=========================================================
+
+This use case explain how to use `ROSbot XL` with `Orbbec Astra` camera to perform autonomous navigation and mapping while taking images of the environment.
```

Comment:
Should be `This use case explains ..."

Reply:
@irenebm is going to open another PR extending this. Please apply this comment there.